dnd.sty

```diff
@@ -153,6 +158,21 @@
   \dndline%
 }
 
+% Fancy First line drop cap for beginning of chapters
+\iftoggle{altdropfont}{% use alternate font
+\RequirePackage{tgothic}
+\newcommand\dndDropFont{\tgothfamily}
+}{% use default
+\input ArtNouvc.fd
+\newcommand*\dndDropFont{\usefont{U}{ArtNouvc}{xl}{n}}
```

Comment:
I created a new dndfonts.sty file for this. It seems simple for a single `newcommand` issue, but if further font refinements happen, it would be the place for it.

Reply:
There already is a dndfonts.sty. Most of the sub-modules are in the `lib` folder.

Reply:
Oof. Ok, using the right file now. Would it also make more sense to define the `DnDDropCapLine` command in `lib/dndfonts.sty` as well?